src/components/forms/Checkbox/Checkbox.tsx

```diff
@@ -36,6 +49,7 @@ export const Checkbox = ({
       />
       <label className="usa-checkbox__label" htmlFor={id}>
         {label}
+        {description}
```

Comment:
instead of needing to deal with nulls for description (which will actually be `undefined` instead of `null` when not passed in) you can do the following:

```suggestion
        {
          labelDescription && 
            <span className="usa-checkbox__label-description">
              {labelDescription}
            </span>
        }
```

That checks whether labelDescription is falsey, and if not renders the span on the other side of the `&&`